crypto/evp/encode.c

```diff
@@ -215,13 +263,13 @@ int EVP_EncodeBlock(unsigned char *t, const unsigned char *f, int dlen)
     return ret;
 }
 
+int EVP_EncodeBlock(unsigned char *t, const unsigned char *f, int dlen)
+{
+    return evp_encodeblock_int(NULL, t, f, dlen);
+}
+
 void EVP_DecodeInit(EVP_ENCODE_CTX *ctx)
 {
-    /* Only ctx->num is used during decoding. */
-    ctx->num = 0;
-    ctx->length = 0;
-    ctx->line_num = 0;
-    ctx->expect_nl = 0;
```

Comment:
Actually, I changed my mind and put it back to zeroing everything explicitly and added a line to additionally zero the flags. The reason is that it occurred to me that some applications might be reusing an EVP_ENCODE_CTX. This would have worked before but not after I removed the explicit zeroing.